src/main.cc

```diff
@@ -94,10 +99,10 @@ file_analysis::Manager* file_mgr = 0;
 zeekygen::Manager* zeekygen_mgr = 0;
 iosource::Manager* iosource_mgr = 0;
 bro_broker::Manager* broker_mgr = 0;
+zeek::Supervisor* zeek::supervisor = 0;
+std::optional<zeek::Supervisor::SupervisedNode> zeek::supervised_node;
```

Comment:
Move this into the Supervisor as a static function returning the node object?

Reply:
Yeah, sounds good.